### DISCUSSION TITLE
How to use different configurations Gambatte

### DISCUSSION THREAD

**Discussion by TheElf01** [OnionUI/Onion#1644] [Q&A]
Hi! new to onionos and miyoo mini

I have a problem i cant solve,  how to use different core options and config for same emulator?  Gambatte

For GB-DMG i want to use a GMB shadder and DMG core options, and for color, a grid shader and gbc options

Thanks!

**Comment by Aemiii91** [comment]
I believe "save content override" is what you're looking for.

https://onionui.github.io/docs/faq#how-do-i-save-retroarch-settings
